### ISSUE TITLE
RuntimeError: await wasn't used with future

### ISSUE THREAD

**Issue by jirkade** [bieniu/brother#143]
Running the example.py script (or using this lib via a home assistant integration) results in an immediate error (I'm not really familiar with python asyncio to try to fix this myself):

```
$ python example.py 172.17.2.17
DEBUG:asyncio:Using selector: EpollSelector
DEBUG:brother:Initializing device 172.17.2.17
Traceback (most recent call last):
  File "/srv/homeassistant/example.py", line 50, in <module>
    loop.run_until_complete(main())
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/srv/homeassistant/example.py", line 31, in main
    brother = await Brother.create(
  File "/srv/homeassistant/lib/python3.9/site-packages/brother/__init__.py", line 95, in create
    await instance.initialize()
  File "/srv/homeassistant/lib/python3.9/site-packages/brother/__init__.py", line 120, in initialize
    _, errstatus, errindex, _ = await hlapi.getCmd(*request_args, *tuple(oids))
RuntimeError: await wasn't used with future
ERROR:asyncio:Task was destroyed but it is pending!
task: <Task pending name='Task-2' coro=<BaseEventLoop.create_datagram_endpoint() running at /usr/lib/python3.9/asyncio/base_events.py:1380> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fbb365fa0>()]>>
ERROR:asyncio:Task was destroyed but it is pending!
task: <Task pending name='Task-3' coro=<AsyncioDispatcher.handle_timeout() running at /srv/homeassistant/lib/python3.9/site-packages/pysnmp/carrier/asyncio/dispatch.py:55> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fbb365f10>()]>>
Exception ignored in: <coroutine object BaseEventLoop.create_datagram_endpoint at 0x7fbb472340>
Traceback (most recent call last):
  File "/usr/lib/python3.9/asyncio/base_events.py", line 1382, in create_datagram_endpoint
  File "/usr/lib/python3.9/asyncio/selector_events.py", line 700, in close
  File "/usr/lib/python3.9/asyncio/base_events.py", line 746, in call_soon
  File "/usr/lib/python3.9/asyncio/base_events.py", line 510, in _check_closed
RuntimeError: Event loop is closed
```

I tried both 2.0.0 and current git with python 3.9 in home assistant virtual environment and 3.11 in a clean Fedora 37 installation, the error is always the same.

**Comment by bieniu** [comment]
Yes, I know about the issue, it is related to `pysnmplib`. I was too quick with merging branch and release new version. Please use version `1.2.3`.

**Comment by bieniu** [comment]
Version `1.2.3` is not compatible with Python 3.11.

**Comment by bieniu** [comment]
And you have to downgrade `pysnmplib`.

**Comment by jirkade** [comment]
Oh, that would be the missing part as 1.2.3 was reporting the same error even with python 3.9.0. Anyway everything works well with pysnmplib 5.0.19.
Thanks.

**Comment by jirkade** [comment]
Yeah, looks like it's the same issue.

**Comment by lionslair** [comment]
![image](https://user-images.githubusercontent.com/832259/206593026-54f085d7-60a7-4734-b151-77e945b254d7.png)

Even after adding that package and the .20 version I still get the same response about invalid host. I know the host is correct.

**Comment by bieniu** [comment]
@lionslair If you have problem with Home Assistant, please open an issue in HA repository. HA is using `brother==2.0.0` and `pysnmplib==5.0.15` and this problem should not accur.

**Comment by bieniu** [comment]
@jirkade I published `brother=2.0.1` with pinned `pysnmplib` version. This version works with Python<3.11.

**Comment by lionslair** [comment]
Sorry. I was just following the links for HA

**Comment by lionslair** [comment]
> @jirkade I published `brother=2.0.1` with pinned `pysnmplib` version. This version works with Python<3.11.

what version of  pysnmplib I am on python 3.10.4

**Comment by lionslair** [comment]
> @jirkade I published `brother=2.0.1` with pinned `pysnmplib` version. This version works with Python<3.11.

bingo its been discovered again
